### PR TITLE
Added back sharebutton to wishlist

### PR DESCRIPTION
**Pull request by AleksandrsKondratjevs**

### CODE REVIEW

File: src/app/component/Header/Header.style.scss [hunk 1]
```diff
@@ -322,14 +322,19 @@
             }
 
             &_edit {
-                width: 70px;
```
Comment (AleksandrsKondratjevs) [inline src/app/component/Header/Header.style.scss:325]: button was not visible, but still had width and moved other objects on mobile in header

File: src/app/component/Header/Header.component.js [hunk 2]
```diff
@@ -142,8 +147,7 @@ export class Header extends NavigationAbstract {
             title: true
         },
         [CART_OVERLAY]: {
-            title: true,
-            edit: true
```
Comment (AleksandrsKondratjevs) [inline src/app/component/Header/Header.component.js:146]: This triggered to render edit button, but I didn't found where it is used in minicart overlay and it just move header items on desktop.

Comment (alfredsgenkins) [reply src/app/component/Header/Header.component.js:146]: Yes, I believe we deprecated it.

File: src/app/component/MyAccountMyWishlist/MyAccountMyWishlist.component.js [hunk 3]
```diff
@@ -187,11 +191,13 @@ export class MyAccountMyWishlist extends PureComponent {
...
         return (
             <button
-              block="Button"
+              mix={ { block: 'MyAccountMyWishlist', elem: 'ShareWishlistButton' } }
               onClick={ shareWishlist }
               disabled={ disabled }
             >
-                { __('Share Wishlist') }
+                <Image
+                  src={ media(SHARE_WISHLIST_ICON, '', false) }
```
Comment (alfredsgenkins) [inline src/app/component/MyAccountMyWishlist/MyAccountMyWishlist.component.js:199]: I think it is more preferable to import this SVG as a resource and simply render it. Like so: https://create-react-app.dev/docs/adding-images-fonts-and-files

File: src/app/component/Header/Header.container.js [hunk 4]
```diff
@@ -45,14 +47,16 @@ export const mapStateToProps = (state) => ({
...
+    activeOverlay: state.OverlayReducer.activeOverlay,
+    isWishlistLoading: state.WishlistReducer.isLoading
 });
 
 /** @namespace Component/Header/Container/mapDispatchToProps */
 export const mapDispatchToProps = (dispatch) => ({
     showOverlay: (overlayKey) => dispatch(toggleOverlayByKey(overlayKey)),
     hideActiveOverlay: () => dispatch(hideActiveOverlay()),
     setNavigationState: (stateName) => dispatch(changeNavigationState(TOP_NAVIGATION_TYPE, stateName)),
+    showPopup: (payload) => dispatch(showPopup(SHARE_WISHLIST_POPUP_ID, payload)),
```
Comment (alfredsgenkins) [inline src/app/component/Header/Header.container.js:59]: I think, this name is misleading, this will trigger the wish-list popup, not just any popup.

**Review by alfredsgenkins — CHANGES_REQUESTED** [review]

**Pull request closed without merging**